### PR TITLE
fix: prompt page translation

### PR DESCRIPTION
**Pull request by imchell**
Before
Incomplete Translation
After
Complete Translation

### CODE REVIEW

**Comment by hsluoyz** [comment]
@imchell plz change all similar texts together, like: https://door.casdoor.com/applications/built-in/app-built-in

image

**Comment by imchell** [comment]
> @imchell plz change all similar texts together, like: https://door.casdoor.com/applications/built-in/app-built-in
> 
> image

translated ✅

File: web/src/locales/zh/data.json [hunk 1]
```diff
@@ -298,6 +302,7 @@
     "Sign in with WebAuthn": "WebAuthn登录",
     "Sign in with {type}": "{type}登录",
     "Signing in...": "正在登录...",
+    "Successfully logged in with webauthn credentials": "Successfully logged in with webauthn credentials",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json]: Not translated

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: web/src/locales/zh/data.json (hunk 1 above)
Comment (hsluoyz) [reply web/src/locales/zh/data.json]: @imchell

Comment (imchell) [reply web/src/locales/zh/data.json]: get it right now

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**